packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -41,33 +62,64 @@ export class CookieStorage implements BaseStorage {
  * or a customized storage. This class is mainly used for commerce-sdk-react library
  * to store authentication tokens.
  */
-export class LocalStorage implements BaseStorage {
-    constructor() {
+export class LocalStorage extends BaseStorage {
+    constructor(options?: BaseStorageOptions) {
+        super(options)
+
         if (typeof window === 'undefined') {
             throw new Error('LocalStorage is not avaliable on the current environment.')
         }
     }
     set(key: string, value: string) {
-        const oldValue = this.get(key)
-        window.localStorage.setItem(key, value)
+        const prefixedKey = this.getPrefixedKey(key)
+        const oldValue = this.get(prefixedKey)
+        window.localStorage.setItem(prefixedKey, value)
         const event = new StorageEvent('storage', {
-            key: key,
+            key: prefixedKey,
             oldValue: oldValue,
             newValue: value
         })
         window.dispatchEvent(event)
     }
     get(key: string) {
-        return window.localStorage.getItem(key) || ''
+        const prefixedKey = this.getPrefixedKey(key)
+        return window.localStorage.getItem(prefixedKey) || ''
     }
     delete(key: string) {
-        const oldValue = this.get(key)
-        window.localStorage.removeItem(key)
+        const prefixedKey = this.getPrefixedKey(key)
+        const oldValue = this.get(prefixedKey)
+        window.localStorage.removeItem(prefixedKey)
         const event = new StorageEvent('storage', {
-            key: key,
+            key: prefixedKey,
             oldValue: oldValue,
             newValue: null
         })
         window.dispatchEvent(event)
     }
 }
+
+export class ServerStorage extends BaseStorage {
+    private map: Map<string, string>
```

Comment:
You can use fancy modern syntax!
```suggestion
    private map = Map<string, string>()
```
(Plus delete line 111)